### PR TITLE
Rewrited eco events

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Rewrited eco-events and added minor fixes
Related to: [927](https://github.com/ita-social-projects/GreenCity/issues/927)

### CODE REVIEW

**Review by nvolodymyr — APPROVED** [review]

**Review by yulkats — APPROVED** [review]

File: src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { NewsDto } from '../../../models/NewsDto';
```
Comment (mplesha) [inline src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts:2]: long path

File: src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+})
+export class EcoEventsItemComponent implements OnInit {
+  @Input() public ecoEvent: NewsDto;
+  @Input() public mainEvent = false;
+  readonly arrow = 'assets/img/icon/arrow.png';
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```
Comment (mplesha) [inline src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts:17]: empty methods and consequently - extra `import` and `implements`

File: src/app/component/home/components/eco-events/eco-events.component.html [hunk 3]
```diff
@@ -4,48 +4,17 @@
       <div class="main-event-image">
         <img [src]="latestNews[0]?.imagePath || eventImg" alt="event-image">
       </div>
```
Comment (mplesha) [inline src/app/component/home/components/eco-events/eco-events.component.html:6]: Why use extra and unnecessary elements? Why those covered constructions, like
```js
<div class='...'>
  <myelement>
  </myelement>
</div>
```
instant of simple constructions :
```
<myelement class=...>
</myement>
```
?

File: src/app/component/home/components/eco-events/eco-events.component.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ export class EcoEventsComponent implements OnInit {
     this.newsService.loadLatestNews();
     this.newsService.latestNews.pipe(catchError(() => of([]))).subscribe(
       (newsItems: NewsDto[]) => {
-        newsItems.forEach(
+        newsItems.map(
           (element: NewsDto) => (element.creationDate = this.convertDate(element.creationDate))
```
Comment (mplesha) [inline src/app/component/home/components/eco-events/eco-events.component.ts:28]: Bad and sadly. It doesn't make sense to replace `forEach` on `map` a formal.  What’s more, you spoiled the functionality.
```js
someArray.forEach(el => el = newElValue);
```
is not equal to the
```js
someArray.map(el => el = newElValue);
```
- check it!

File: src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts (hunk 1 above)
Comment (RomanDeveloperAcc) [reply src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts]: Fixed

File: src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts (hunk 2 above)
Comment (RomanDeveloperAcc) [reply src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts]: Fixed

File: src/app/component/home/components/eco-events/eco-events.component.html (hunk 3 above)
Comment (RomanDeveloperAcc) [reply src/app/component/home/components/eco-events/eco-events.component.html]: Fixed

File: src/app/component/home/components/eco-events/eco-events.component.ts (hunk 4 above)
Comment (RomanDeveloperAcc) [reply src/app/component/home/components/eco-events/eco-events.component.ts]: Fixed

**Review by HryshkoAnton — APPROVED** [review]

**Review by sofiiamatsyhin — APPROVED** [review]

**Comment by mplesha** [comment]
Conflicting files

File: tsconfig.json [hunk 5]
```diff
@@ -27,6 +27,7 @@
       "@eco-news-service/*": ["src/app/component/eco-news/services/*"],
       "@eco-news-store/*": ["src/app/component/eco-news/store/*"],
       "@global-errors/*": ["src/app/component/errors/*"],
+      "home-models/*": ["src/app/component/home/models/*"],
```
Comment (mplesha) [inline tsconfig.json:30]: use `@` for path aliases

**Pull request merged**